internal/cli/atlas/quickstart/confirm_cluster_setup.go

```diff
@@ -53,12 +53,14 @@ Cluster Disk Size (GiB):		%.1f`, opts.Tier, diskSize)
 [Confirm cluster settings]
 Cluster Name:				%s%s
 Cloud Provider and Region:		%s
+Cluster Termination Protection:		%t
```

Comment:
@dianchenghu @boooczek This is how the confirmation message looks (I highlighted the message for cluster termination protection):
<img width="380" alt="Screenshot 2023-01-29 at 20 45 13" src="https://user-images.githubusercontent.com/29061011/215354848-563b43d4-0ce0-4729-9390-594a9e891682.png">

I've seen that for "Load sample data" we only show it if the user has set the flag to `true`, should we do the same thing for cluster termination protection? Or should we always display it, even if it was set to false? And if so, are boolean values ok (true/false) as answers, or should we change it to Yes/No?

Reply:
My opinion: Lets hide when "false". Also it should be consistent with other questions of this type so should be Yes / No.

I'm also not sure about the flag --enableTerminationProtection in --help. If they want to disable they'd do --disableTerminationProtection? 
On that --help please also update the copy slightly to say "Users can't delete a cluster..."

Reply:
Ok, will update the confirmation message.

By default termination protection is off when creating a cluster, so we would only need a flag that enables it

Reply:
Agree with hide when "false", since the quickstart target users might not need to worry about data protection at this stage.  For the "true/false" copy problem, I think we could change the label to "Enable Cluster Termination Protection: Yes" which might be more consistent with the flag.

Reply:
When termination protection is disabled (the message is hidden)
<img width="368" alt="Screenshot 2023-01-30 at 15 35 38" src="https://user-images.githubusercontent.com/29061011/215521693-2c1b708b-47a6-4b90-b4c4-40be31b1fbd1.png">

When termination protection is enabled through the flag:
<img width="342" alt="Screenshot 2023-01-30 at 15 37 25" src="https://user-images.githubusercontent.com/29061011/215522098-897d5bf2-445f-419e-8803-d3ebb04ba161.png">

When termination protection flag is not specified, and started creating a cluster through the interactive setup:
<img width="416" alt="Screenshot 2023-01-30 at 15 38 40" src="https://user-images.githubusercontent.com/29061011/215522417-4374aecf-07b9-4f7d-b7ef-6be72e774cb9.png">

Reply:
The first two images look good to me! For the last one, I'm not sure if that additional step is required, but I won’t suggest adding it to the interactive flow. Could you confirm with @boooczek  on it?

Reply:
>"Users can't delete a cluster..."

FYi the original copy is based on docs style guides, which favor active voice while the current change makes it more passive

Reply:
Discussed offline: We don't want to alter the interactive setup process any more - it should only ask for options that are crucial to setting up a cluster. Therefore, we will be reverting any prompts related to cluster termination, but keep the flag.